.github/workflows/merge.yaml

```diff
@@ -0,0 +1,40 @@
+on: [push, pull_request]
+
+name: Quality assurance
+
+jobs:
+    merge:
+        name: "Auto-Merge"
+
+        runs-on: "ubuntu-latest"
+
+        needs:
+            - "php-cs-fixer"
+            - "composer-normalize"
+            - "phpstan"
```

Comment:
@localheinz does this work, if we have defined these jobs in seperate workflows?

Reply:
No it won't work. I fixed this by having only one workflow file containing all jobs in #770

Reply:
👍